topo_processor/cli/geostore/delete.py

```diff
@@ -0,0 +1,45 @@
+import boto3
+import click
+from linz_logger import LogLevel, get_log, set_level
+
+from topo_processor.geostore.invoke import invoke_lambda
+from topo_processor.util.time import time_in_ms
+
+
+@click.command()
+@click.option(
+    "-id",
+    "--dataset-id",
+    required=True,
+    help="The dataset id to delete",
+)
+@click.option(
+    "-p",
+    "--prod",
+    is_flag=True,
+    help="Use this flag to delete into production environment",
+)
+@click.option(
+    "-v",
+    "--verbose",
+    is_flag=True,
+    help="Use verbose to display trace logs",
+)
+def main(dataset_id: str, prod: bool, verbose: str) -> None:
+    start_time = time_in_ms()
+    get_log().info("delete_datasets_start", dataset_id=dataset_id, isProduction=prod)
+
+    if verbose:
+        set_level(LogLevel.trace)
+
+    client = boto3.client("lambda")
+
+    try:
+        delete_parameters = {"id": dataset_id}
+        response = invoke_lambda(client, "datasets", "DELETE", delete_parameters, prod)
+        if response["status_code"] != 204:
```

Comment:
generally checking for a specific 200 status code is not a good idea, is there some sort of `response.ok`? which would be `status_code >=200 && <= 299`

Reply:
The response when a delete is successful is `{"status_code": 204, "body": {}}` so I don't know what to do with that?

Reply:
I would make a is_ok function that checks that status is in the 200 range